gdk/Display.custom

```diff
@@ -75,12 +75,8 @@
 
 		public void AddClientMessageFilter (Gdk.Atom message_type, Gdk.FilterFunc func) 
 		{
-			GdkSharp.FilterFuncWrapper func_wrapper = new GdkSharp.FilterFuncWrapper (func);
-			if (!PersistentData.Contains ("client_message_filter_func_list"))
-				PersistentData ["client_message_filter_func_list"] = new ArrayList ();
-			ArrayList func_list = PersistentData ["client_message_filter_func_list"] as ArrayList;
-			func_list.Add (func_wrapper);
-			gdk_display_add_client_message_filter (Handle, message_type == null ? IntPtr.Zero : message_type.Handle, func_wrapper.NativeDelegate, IntPtr.Zero);
+			var gch = GCHandle.Alloc (func);
+			gdk_display_add_client_message_filter (Handle, message_type == null ? IntPtr.Zero : message_type.Handle, GdkSharp.FilterFuncWrapper.NativeDelegate, (IntPtr)gch);
```

Comment:
Can we add a comment that this leaks forever and that there is no `gdk_display_remove_client_message_filter` to allow cleanup to happen?

Reply:
Done.